### ISSUE TITLE
Does Ditto support mimic typing ? (simulate key

### ISSUE THREAD

**Issue by blogmotion** [sabrogden/Ditto#266]
Hello,

Everything in the title : how can I mimic typing instead of traditional paste
I need this to inject clipboard content in a remote Window where paste is not working (accross multiple remote desktop)

The workaround is to use keepass with the autotype but it is tedious and not fast.

([Paste Texte Like](https://sites.google.com/site/pastetextlike/) and [ArsClip](http://www.joejoesoft.com/cms/showpage.php?cid=97) both support this feature)

Thank you

**Comment by navossoc** [comment]
I'm trying to find the same information, so far my conclusion is this feature doesn't exist.

I was looking for an extra softwares that can "auto type" for you, but most of them require to bind a hotkey, insert the text, etc... It's not as fast as if Ditto type for us the content of the clipboard.

I would like to see this feature too...

**Comment by garoto** [comment]
I'm assuming theis is what you guys are looking for? 

https://github.com/sabrogden/Ditto/wiki/Custom-Key-Strokes

```
> reg query HKCU\Software\Ditto\PasteStrings

HKEY_CURRENT_USER\Software\Ditto\PasteStrings
    gvim.exe    REG_SZ    "{PLUS}gP
    kitty.exe    REG_SZ    {Delay 50}+{INS}
    putty.exe    REG_SZ    {Delay 50}+{INS}
    xpcmd.exe    REG_SZ    % {Delay 50}ep
    putty-d2d.exe    REG_SZ    {Delay 50}+{INS}
    ssh.exe    REG_SZ    {Delay 50}^+p
    console.exe    REG_SZ    {Delay 50}+{INS}
    cmd.exe    REG_SZ    % {Delay100}ep
```

**Comment by navossoc** [comment]
@garoto Kinda, but not exactly...
In this way it can't type each key.

Think in a webpage with a form with an input with some kind of "paste protection", you need to type each letter.

You can't achieve that with Custom Key Strokes since the value need to be hardcoded, for example:

```
{VKEY P}{VKEY A}{VKEY S}{VKEY S}{VKEY W}{VKEY O}{VKEY R}{VKEY D}
```
(not a valid example, I know, you need the keycodes, but you get it)

This would be nice if it can work with the data you on the clipboard.

If there is a way to do that, I didn't see, sorry.

[]'s

**Comment by blogmotion** [comment]
To conclude it's not possible to do this with Ditto. I found a way to do this with KeePass and _{CLIPBOARD}_ autotype value

**Comment by vm75** [comment]
This feature is available in Arsclip. That used to be my favorite clipboard tool till I discovered Ditto. Now I use that only for pasting to browser. 
+1 for this feature request.

**Comment by Chealer** [comment]
@blogmotion: No, it does not, as reported in ticket #100. Note that this is an ITS, not a discussion forum.

**Comment by garoto** [comment]
I've being working around this limitation by setting an option in the options and literally hitting `ctrl+c` in Ditto's window to copy highlighted entry to the the clipboard without the auto-paste mechanic. Then I proceed to paste manually in troublesome input dialogs.
